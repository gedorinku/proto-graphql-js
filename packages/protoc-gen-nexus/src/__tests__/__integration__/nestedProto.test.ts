import { testSchemaGeneration } from "../__helpers__/process.test.helper";

testSchemaGeneration("nested", "protobufjs");
