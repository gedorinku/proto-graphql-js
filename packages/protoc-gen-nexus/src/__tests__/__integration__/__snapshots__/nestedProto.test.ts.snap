// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schema generation with protobufjs can make GraphQL schema from generated DSLs: schema.graphql 1`] = `
"### This file was generated by Nexus Schema
### Do not make changes to this file directly


\\"\\"\\"\\"\\"\\"
type ParentMessage {
  \\"\\"\\"\\"\\"\\"
  body: String!

  \\"\\"\\"\\"\\"\\"
  nested: ParentMessageNestedMessage

  \\"\\"\\"\\"\\"\\"
  nestedEnum: ParentMessageNestedEnum
}

\\"\\"\\"\\"\\"\\"
input ParentMessageInput {
  \\"\\"\\"\\"\\"\\"
  body: String!

  \\"\\"\\"\\"\\"\\"
  nested: ParentMessageNestedMessageInput

  \\"\\"\\"\\"\\"\\"
  nestedEnum: ParentMessageNestedEnum
}

\\"\\"\\"\\"\\"\\"
enum ParentMessageNestedEnum {
  \\"\\"\\"\\"\\"\\"
  BAR

  \\"\\"\\"\\"\\"\\"
  FOO
}

\\"\\"\\"\\"\\"\\"
type ParentMessageNestedMessage {
  \\"\\"\\"\\"\\"\\"
  nestedBody: String!
}

\\"\\"\\"\\"\\"\\"
input ParentMessageNestedMessageInput {
  \\"\\"\\"\\"\\"\\"
  nestedBody: String!
}

type Query {
  ok: Boolean!
}
"
`;
